wgengine/router/router_userspace_bsd.go

```diff
@@ -101,9 +102,39 @@ func inet(p netaddr.IPPrefix) string {
 	return "inet"
 }
 
+// See https://github.com/tailscale/tailscale/issues/1307#issuecomment-786045280
+// Remove all IPv6 entries.
+func (r *userspaceBSDRouter) modifyConfigForFreeBSDBugWorkaround(cfg *Config) {
```

Comment:
I actually forget now whether it's safe/allowed to mutate the Config passed to Set.

to not have to worry about it you can return a *Config copy here instead and make the caller do:

```go
       cfg = r.modifiedConfigForFreeBSDBugWorkaround(cfg)
```

(note rename to "modified Config")

Reply:
Done.